test/integration/thruster.cc

```diff
@@ -285,14 +314,50 @@ void ThrusterTest::TestWorld(const std::string &_world,
       {
         EXPECT_NEAR(0.0, angVel.X(), _baseTol);
       }
+      else if (_namespace == "deadband")
+      {
+        continue;
+      }
       else
       {
-        EXPECT_NEAR(omega, angVel.X(), omegaTol) << i;
+        ASSERT_NEAR(omega, angVel.X(), omegaTol) << i;
       }
     }
     EXPECT_NEAR(0.0, angVel.Y(), _baseTol);
     EXPECT_NEAR(0.0, angVel.Z(), _baseTol);
   }
+
+  modelPoses.clear();
+  propellerAngVels.clear();
+  propellerLinVels.clear();
+  // Make sure that when the deadband is disabled
+  // commands below the deadband should create a movement
+  auto latest_pose = modelPoses.back();
+  msgs::Boolean db_msg;
+  if (_namespace == "deadband")
+  {
+    // disable the deadband
+    db_msg.set_data(false);
+    db_pub.Publish(db_msg);
+    // And we send a command that is still below the deadband threshold
+    msg.set_data(_deadband / 2.0);
+    pub.Publish(msg);
+    // Check movements
+    fixture.Server()->Run(true, 1000, false);
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    EXPECT_LT(0.1, modelPoses.back().Pos().X());
+
+    EXPECT_EQ(1000u, modelPoses.size());
+    EXPECT_EQ(1000u, propellerAngVels.size());
+    EXPECT_EQ(1000u, propellerLinVels.size());
+    modelPoses.clear();
+    propellerAngVels.clear();
+    propellerLinVels.clear();
+  }
+  // re-enable the deadband
+  db_msg.set_data(true);
+  db_pub.Publish(db_msg);
+
```

Comment:
And then what does this achieve? We are not checking any assertions here.

Reply:
that is indeed not necessary. I'll remove those lines in the next commit

Reply:
After giving it some thoughts, one reason I had this at the end of the test, was in case someone wants to extend the test and forgot to re-enable the deadband. But I guess, its the tester responsibility to make sure they are in the conditions they want. So I'll remove it anyway.